### DISCUSSION TITLE
Have any one got seleniumbase working on a Raspberry?

### DISCUSSION THREAD

**Discussion by valnurat** [seleniumbase/SeleniumBase#2864] [Q&A] [answered]
I'm trying to do a webscraper and I have succeeded on windows with Selenium and undetected chromedriver. I then wanted to move it to my raspberry, but undetected chromedriver gives issues. So now I'm trying to find another solution and I now I want to have SB to work on my Raspberry with Debian GNU/Linux 12 (bookworm). And ofcourse I tried to look into the descussions if other had issues with SB on a raspberry. I found this post that was answered from @mdmintz for getting undetected chromedriver to work. I followed the answer [https://github.com/seleniumbase/SeleniumBase/discussions/2286](url) and it looked easy. Download ,[chromedriver-linux64.zip](https://storage.googleapis.com/chrome-for-testing-public/126.0.6478.63/linux64/chromedriver-linux64.zip), renamed it and copied it to the seleniumbase/drivers folder.

What I have now in my seleniumbase/drivers is this:
![image](https://github.com/seleniumbase/SeleniumBase/assets/76236110/7b327367-5bce-491f-9c48-a2024512be81)

I then just wanted a very basic test code, so I did this and when I run the code nothing happens.
![image](https://github.com/seleniumbase/SeleniumBase/assets/76236110/1a12da78-8694-451e-ad8d-1dc730b2d295)

What do I miss?
Br

**Comment by mdmintz** [comment, marked as answer]
Your example uses the `BaseCase` format ([help_docs/syntax_formats.md](https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/syntax_formats.md)) which runs with `pytest`, instead of `python`.
Add `BaseCase.main(__name__, __file__)` to trigger `pytest` when running that file with `python`.
